### ISSUE TITLE
What does ListOnSteam and ListOnEOS do?

### ISSUE THREAD

**Issue by IceOfWraith** [StunlockStudios/vrising-dedicated-server-instructions#135]
Our community has noticed a lot of issues with today's release and it seems to be related to the fact that ListOnSteam was defaulted to false.

It seems setting ListOnSteam to true fixed their issue, but we are unsure what ListOnEOS entails. We know EOS is the Epic network, but what is required to set it up and can both be enabled at the same time?

If you can please update the guide it would be appreciated! :)

**Comment by IamDontNoob** [comment]
Does anyone know the commands for Overrides  ListOnSteam and ListOnEOS in commandline parameters?

For example ListOnMasterServer used
Setting: ListOnMasterServer
Description: Register on list server or not.
Type: boolean
Example value: true, false
Overrides: VR_LIST_ON_MASTER_SERVER=<value>, -listing <value>, -enableListing, -disableListing


I tied -ListOnSteam true, -enableListOnSteam, -listingOnSteam, -enableListingOnSteam but it's does not work

**Comment by kottik-mypp** [comment]
Config from https://steamcommunity.com/app/1604030/discussions/0/3832045251550037194/ work for me

**Comment by IceOfWraith** [comment]
Indeed. I discovered the ListOnSteam does Steam and ListOnEOS does in-game. ListOnMasterServer is deprecated.

Now to figure out what the API setting is for.
